dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventsExportControllerTest.java

```diff
@@ -269,6 +296,76 @@ void getEventByIdWithDataValues() {
     assertHasMember(dataValue, "storedBy");
   }
 
+  @ParameterizedTest
+  @ValueSource(
+      strings = {
+        "in:red",
```

Comment:
```suggestion
        "in:(red,yellow)",
```
Could we trying some more cases here?
I am not sure myself what is the expected behavior if we use more values in the in clause and only some of them match some of the multitext data value